azureiot.sln

```diff
@@ -95,7 +95,11 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Devices.Shared.Tests", "shared\tests\Microsoft.Azure.Devices.Shared.Tests.csproj", "{CEEE435F-32FC-4DE5-8735-90F6AC950A01}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Devices.PnpServiceSample", "iothub\service\samples\PnpServiceSamples\Thermostat\Microsoft.Azure.Devices.PnpServiceSample.csproj", "{5B7C14D6-1441-489B-B9FB-1EEEA43B8C48}"
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "samples", "samples", "{810FAC63-974E-4B5A-8E56-4B95FA0E3FAE}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TemperatureController", "iothub\service\samples\PnpServiceSamples\TemperatureController\TemperatureController.csproj", "{66B7F96C-249D-4F26-9266-D9B270E5F96D}"
```

Comment:
I would have liked to see the sample folder structure be: samples -> PnpDeviceSample/ PnpServiceSample -> Thermostat, TemperatureController.

That way, although there is an additional layer there, we clearly mark that these samples added are pnp related.
What do you think?